packages/mock-repo/src/composite/composite.tsx

```diff
@@ -7,7 +7,17 @@ export interface IProps {
   className?: string;
 }
 
-export const Composite: React.SFC<IProps> = (props) =>
-  <div {...style('root', {}, props)}><ChildComp text={props.text} /></div>;
+export interface IState {
+  text?: React.ReactNode;
+}
+
+export class Composite extends React.Component<IProps, IState> {
+  public static displayName: string;
+  public state: IState = {text: ''};
+
+  public render() {
```

Comment:
I think we need an example that's a bit more "realistic", like using `setState` in the comp

Reply:
Something like a counter that increments on each click. Or a dropdown with `open` state.